### ISSUE TITLE
Missing functionality: Undecorated window resizing in `dioxus-desktop`

### ISSUE THREAD

**Issue by chipnertkj** [DioxusLabs/dioxus#3128]
## Feature Request
Disabling window decorations in `tao` when using `dioxus-desktop` prevents the window from being resized, as confirmed in https://github.com/tauri-apps/tao/issues/994#issuecomment-2442048551.
`dioxus` does not provide the required API to bring drag-resizing back to a window without decorations.
This behavior prevents valid use cases like custom title bars. The user has to implement a lot of boilerplate for the webview to be resizable again.
`tauri` has their own implementation - I think `dioxus` should too.

See https://github.com/tauri-apps/tauri/issues/7900#issuecomment-1734677770 for an example of the behavior that appears when disabling a window's decorations and attempting to resize.
Note: the video only showcases attempts made at the upper-right corner. On my end, I am also completely unable to resize the window unless im dragging from the upper edge of the window, in which case it is still difficult to grab onto - the area is one pixel high.

## Implement Suggestion

`tauri` implements this here: [src/undecorated_resizing.rs](https://github.com/tauri-apps/tauri/blob/dev/crates/tauri-runtime-wry/src/undecorated_resizing.rs)

**Comment by iKineticate** [comment]
I completely agree with this suggestion. I’m also using Dioxus and encountered the same issue with window resizing when disabling window decorations. Without the proper API to enable drag-resizing on undecorated windows, it makes it difficult to implement features like custom title bars, as mentioned in the issue.

**Comment by LikeCrazyShawn** [comment]
hi, may I know if this issue is fixed ? and which version if yes?

**Comment by keiko37** [comment]
> hi, may I know if this issue is fixed ? and which version if yes?

Also have this issue. As I see it hasn't been fixed yet.

**Comment by LikeCrazyShawn** [comment]
can we prioritize this for near release? Undecorated window feature is important for people to build professional looking apps

**Comment by safwa1** [comment]
hi, may I know if this issue is fixed ? and which version if yes?
